pkg/serverless/trace/span_modifer_test.go

```diff
@@ -63,7 +65,9 @@ func TestInferredSpanFunctionTagFiltering(t *testing.T) {
 	agnt.ModifySpan = spanModifier.ModifySpan
 	defer cancel()
 
-	tp := testutil.TracerPayloadWithChunk(testutil.RandomTraceChunk(2, 1))
+	tc := testutil.RandomTraceChunk(2, 1)
+	tc.Priority += 1 // ensure trace is never sampled out
+	tp := testutil.TracerPayloadWithChunk(tc)
```

Comment:
These tests were failing because the traces created with `testutil.RandomTraceChunk` were sometimes being sampled out.

Solution is to increase the priority of the random traces to ensure they are never sampled out.